backend/climateconnect_main/celery.py

```diff
@@ -12,3 +13,10 @@
 #   should have a `CELERY_` prefix.
 app.config_from_object('django.conf:settings', namespace='CELERY')
 app.autodiscover_tasks()
+
+app.conf.beat_schedule = {
+    'schedule_automated_email_reminder_for_notifications': {
+        'task': 'climateconnect_api.tasks.schedule_automated_reminder_for_user_notifications',
+        'schedule': timedelta(days=2)  # Task runs every two days
```

Comment:
Should we only run this every 2 days? This would mean that if you get a message right after the task ran it would take almost 4 days until you get a reminder. Maybe instead we could log when we sent reminders in the db and run this task more often? Then we could also track better how many reminders we're sending.

Reply:
We can run this task everyday, but I don't think so we should run this task more than once a day.

Reply:
Is that so that we don't send too many emails? Because for when to send an email I'd just save in the db when we last sent one. But sure, once per day seems good!

Reply:
Changed this!